Gemfile

```diff
@@ -2,7 +2,7 @@ source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
 group :test do
   gem 'rake', '~> 10.4'
-  gem 'puppet', ENV['PUPPET_VERSION'] || '~> 3.8'
+  gem 'puppet', ENV['PUPPET_VERSION'] || '~> 4.0'
```

Comment:
Oh, it isn't really. That's just what I've been using for testing. All the tests should be specifying PUPPET_VERSION, though so it seems like a sensible default to me?